src/langcheck/augment/_common/_jailbreak_template.py

```diff
@@ -69,9 +84,13 @@ def jailbreak_template_common(
             selected_templates = templates
 
         for template_name in selected_templates:
-            template = get_template(
-                f"{language}/jailbreak_templates/{template_name}.j2"
-            )
+            # if the template is from custom_templates, fetch the path from there
+            if custom_templates is not None and any(name == template_name for name, _ in custom_templates):
+                template = get_template(next(path for name, path in custom_templates if name == template_name))
```

Comment:
This `get_template` function is made for access with relative path from our internal `metrics/prompts/_utils.py`. (Ref: https://github.com/matchcase/langcheck/blob/9747eb85ff895d364fd7db41120ed2f054510f2b/src/langcheck/metrics/prompts/_utils.py#L9)

So if you now specify the file with a relative path, this `get_template` fails while it passes the validation.

Could you load the template directly as `custom_evaluator` does here? It would be also helpful if you could do some refactoring, but no need if you don't want to!!
https://github.com/matchcase/langcheck/blob/9747eb85ff895d364fd7db41120ed2f054510f2b/src/langcheck/metrics/custom_text_quality.py#L102